src/main/java/seedu/teachstack/logic/commands/CommandResult.java

```diff
@@ -79,4 +99,16 @@ public String toString() {
                 .toString();
     }
 
+    public boolean isShowPopUp() {
+        return showPopUp;
+    }
+
+    public Node getAdditionalComponent() {
+        return additionalComponent; // Getter for the additional UI component
+    }
+
+
+
+
+
```

Comment:
Slight nitpick but perhaps these lines could be removed?

Reply:
resolved in latest commit